index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />    <link rel="stylesheet" href="style.css">
+    <title>Cars-connect-254 conference website</title>
+</head>
+<body>
+    <div class=" wrapper">
+            <nav class="topper">
+                <ul class="top-nav">
+                    <li><i class="fa-brands fa-facebook-f"></i></li>
+                    <li><i class="fa-brands fa-twitter"></i></li>
+                    <li><a href="#"></a>English</li>
+                    <li><a href="#"></a>MyPage</li>
+                    <li><a href="#"></a>Logout</li>
+                </ul>
+            </nav>
+            <nav class="main-navbar">
+                
+                <div class="hamburger-menu">
+                    <div id="loggo">
+                        <a href="index.html"><img src="images/logo.png" alt="logo" id="logo"></a>
+                    </div>
+                    
+                    <input id="menu__toggle" type="checkbox" />
+                    <label class="menu__btn" for="menu__toggle">
+                        <span></span>
+                    </label>
+                    
+                        
+                    
+                        <ul class="menu__box" >
+                            <li><a class="menu__item" href="about.html">About</a></li>
+                            <li><a class="menu__item" href="#">Program</a></li>
+                            <li><a class="menu__item" href="#">Sponser</a></li>
+                            <li><a class="menu__item" href="#">Join</a></li>
+                            <li><a class="menu__item" href="#">News</a></li>
+                            <li id="campaign"><a class="menu__item" href="#">Campaign</a></li>
+                        </ul>
+
+                    </div>
+                    
+                </div>
+
+            </nav>
```

Comment:
- [ ] The menu items on the first part and the second part on your header is not properly aligned on the right as we have it on the design template. Take a look at below screenshots and walk your design alongside the guiding template.

### Your Design

![your-header-align](https://user-images.githubusercontent.com/26597432/226090474-904d02da-cace-42bb-a02b-807fbe77c1ca.png)

### Guiding Design
![temp-header-align](https://user-images.githubusercontent.com/26597432/226090465-7295591e-4a2c-4967-9ecd-840fa85e79b9.png)